backend/StreamServerApp/views/subtitles.py

```diff
@@ -0,0 +1,36 @@
+
+from rest_framework import viewsets, generics
+from StreamServerApp.serializers.subtitles import SubtitleListSerializer
+from StreamServerApp.models import Subtitle
+from StreamServerApp.media_processing import convert_subtitles_to_webvtt
+import os
+from django.conf import settings
+
+
+class SubtitleViewSet(viewsets.ModelViewSet):
+    """
+    This viewset automatically provides `list` and `search` actions for Videos
+    """
+
+    queryset = Subtitle.objects.all()
+    serializer_class = SubtitleListSerializer
+
+    def _allowed_methods(self):
+        return ['GET', 'POST']
+
+    def perform_create(self, serializer):
```

Comment:
Would it be possible to pass to serializer.save some extra arg to only have to perform one db operation here instead of two?